src/bootstrap/doc.rs

```diff
@@ -317,7 +317,8 @@ fn invoke_rustdoc(builder: &Builder, compiler: Compiler, target: Interned<String
         .arg("-o").arg(&out)
         .arg(&path)
         .arg("--markdown-css")
-        .arg("../rust.css");
+        .arg("../rust.css")
+        .arg("--generate-redirect-pages");
```

Comment:
This doesn't do anything because this code is only used for markdown files in `src/doc/book/redirects`.